src/SBarcodeDecoder.cpp

```diff
@@ -190,15 +199,70 @@ QImage SBarcodeDecoder::videoFrameToImage(QVideoFrame &videoFrame, const QRect &
         return image.rgbSwapped().copy(captureRect);
     }
 
+    #else
+
+    auto handleType = videoFrame.handleType();
+
+    qDebug() << handleType;
```

Comment:
While useful during debugging I don't think we should produce any console output in our library unless some special would be set. Maybe you can add something like `ENABLE_DEBUG_LOGS` to switch at compile time?

Reply:
No need anymore, will be deleted